flink-python/pyflink/table/tests/test_environment_completeness.py

```diff
@@ -41,6 +41,7 @@ def excluded_methods(cls):
             'getCompletionHints',
             'fromValues',
             'create',
+            'createTemporaryTable',
```

Comment:
Is it worth exposing these through PyFlink as well? If I understand correctly that requires replicating the (Table/Format)Descriptor classes in Python, right?

If we want to do this I would prefer opening another sub-task for the FLIP where we expose all methods for PyFlink since it'll mostly be pass-through work.

Reply:
Yes, at some point it would be good to have this in Python as well. But maybe someone from the Python team can pick up this subtask. Let's focus on Java now. Once those interfaces have stabilitized in the next days/weeks, somebody can copy them to Python.

Reply:
I'll add a sub-task to the umbrella issue then.